build.sbt

```diff
@@ -68,7 +68,7 @@ val boundedCore = (project in file("bounded-core"))
   .settings(basicSettings: _*)
   .settings(
     name := "bounded-core",
-    libraryDependencies ++= Dependencies.baseDeps)
+    libraryDependencies ++= Dependencies.baseDeps ++ Dependencies.persistanceLmdbDBDeps)
```

Comment:
Hmm, this triggers a thought: Now we include dependencies for *all* supported stores (i.e. cassandra, lmdb, etc.) while an application only needs 1. If we don't want this we should disconnect the vendor implementation of bounded from core, and make separate packages of them.... For now maybe overkill, but worthwhile to create an issue for this?

Reply:
small typo in the dependency name: persistenceLmdbDBDeps